app/tests/helpers/example_usage.md

```diff
@@ -66,7 +67,7 @@ import compiledProjectContactFormQuery, {
 
 // The query mimics a page that contains that component,
 // we craft a test query that uses the fragments of the component we're testing.
-const loadedQuery = graphql`
+const testQuery = graphql`
```

Comment:
Different examples used different variable names for this; I went with `testQuery` for everything because that's what's in the class